requirements.txt

```diff
@@ -734,13 +741,13 @@ mkdocs-material-extensions==1.1.1 \
     --hash=sha256:9c003da71e2cc2493d910237448c672e00cefc800d3d6ae93d2fc69979e3bd93 \
     --hash=sha256:e41d9f38e4798b6617ad98ca8f7f1157b1e4385ac1459ca1e4ea219b556df945
     # via mkdocs-material
-mozilla-metric-config-parser==2023.9.1 \
```

Comment:
this was using `mozilla-nimbus-schemas==2023.6.6` which does not exist

Reply:
Strange! It looks like it exists in pypi https://pypi.org/project/mozilla-nimbus-schemas/2023.6.6/

Reply:
Oh this is the version that was broken and yanked. It should've been allowed per my understanding of a yanked release, but oh well. Still better to be up-to-date.